possum/pipeline_validation/models.py

```diff
@@ -0,0 +1,100 @@
+    sbid = models.BigIntegerField(db_column='sbid')
+    tile1 = models.ForeignKey('survey.Tile', models.DO_NOTHING, db_column='tile1', to_field='tile', related_name='band1_tile1')
+    tile2 = models.ForeignKey('survey.Tile', models.DO_NOTHING, blank=True, null=True, db_column='tile2', to_field='tile', related_name='band1_tile2')
+    tile3 = models.ForeignKey('survey.Tile', models.DO_NOTHING, blank=True, null=True, db_column='tile3', to_field='tile', related_name='band1_tile3')
+    tile4 = models.ForeignKey('survey.Tile', models.DO_NOTHING, blank=True, null=True, db_column='tile4', to_field='tile', related_name='band1_tile4')
+    type = models.CharField(choices=PARTIAL_TILES_TYPE, db_column='type')
+    number_sources = models.PositiveIntegerField(blank=True, null=True, db_column='number_sources')
+    _1d_pipeline = models.CharField(choices=PIPELINE_STATE, db_column='1d_pipeline', blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'partial_tile_1d_pipeline_band1'
+        unique_together = (('observation', 'sbid', 'tile1', 'tile2', 'tile3', 'tile4', 'type'),)
+
+class PartialTilePipelineRegionsBand2(models.Model):
+    id = models.BigAutoField(primary_key=True, db_column='id')
+    observation = models.ForeignKey('survey.Observation', models.DO_NOTHING, db_column='observation', to_field='name')
+    sbid = models.BigIntegerField(db_column='sbid')
+    tile1 = models.ForeignKey('survey.Tile', models.DO_NOTHING, db_column='tile1', related_name='band2_tile1')
+    tile2 = models.ForeignKey('survey.Tile', models.DO_NOTHING, blank=True, null=True, db_column='tile2', related_name='band2_tile2')
+    tile3 = models.ForeignKey('survey.Tile', models.DO_NOTHING, blank=True, null=True, db_column='tile3', related_name='band2_tile3')
+    tile4 = models.ForeignKey('survey.Tile', models.DO_NOTHING, blank=True, null=True, db_column='tile4', related_name='band2_tile4')
+    type = models.CharField(choices=PARTIAL_TILES_TYPE, db_column='type')
+    number_sources = models.PositiveIntegerField(blank=True, null=True, db_column='number_sources')
+    _1d_pipeline = models.CharField(choices=PIPELINE_STATE, db_column='1d_pipeline', blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'partial_tile_1d_pipeline_band2'
+        unique_together = (('observation', 'sbid', 'tile1', 'tile2', 'tile3', 'tile4', 'type'),)
+
+class ObservationBand1_1DPipelineValidation(models.Model):
+    name = models.OneToOneField('survey.Observation', models.DO_NOTHING, db_column='name', to_field='name', primary_key=True)
+    sbid = models.BigIntegerField(blank=True, null=True, db_column='sbid')
+    _1d_pipeline_validation = models.CharField(blank=True, null=True, db_column='1d_pipeline_validation')
+    single_SB_1D_pipeline = models.CharField(blank=True, null=True, db_column='single_sb_1d_pipeline')
+    comments = models.TextField(blank=True, null=True, db_column='comments')
```

Comment:
These two could be added to the observation table as new columns

Reply:
They were initially in 1 table for band 1 and band 2, but Erik said they should be in separate tables as the footprints are different and there'd be a lot of empty columns.
e.g. if it was:
observation (EMU_123, 1d_pipeline_validation_band1, single_SB_validation_band1, 1d_pipeline_validation_band2, single_SB_validation_band2), if the band1 columns are present, the band2 ones would be empty, vice versa.